src/gulp-tasks-linters.js

```diff
@@ -17,13 +17,21 @@ export function linterTasks (gulp, opts) {
     scssLintPath = path.resolve(__dirname, '../.scss-lint.yml');
   }
 
-  let esLintPath = path.resolve(process.cwd(), '.eslintrc');
+  let esLintPath = options.eslintConfigPath || path.resolve(process.cwd(), '.eslintrc');
   try {
     fs.accessSync(esLintPath, fs.F_OK);
   } catch (e) {
     esLintPath = path.resolve(__dirname, '../.eslintrc');
   }
 
+  let eslintOverride = options.eslintOverride ? 
+    require(options.eslintOverride) : {};
+
+  if (options.customEslintPath) {
+    eslintOverride = require(options.customEslintPath);
+    console.warn('customEslintPath has been deprecated. You should use eslintOverride instead');
+  }
+
   const customEslint = options.customEslintPath ?
```

Comment:
I believe this line can be removed? From what I see we are not using customEslint anymore.